### DISCUSSION TITLE
Stepping down from QTermWidget and QTerminal maintenance

### DISCUSSION THREAD

**Discussion by yan12125** [lxqt/lxqt#2564] [General]
I haven't used qterminal as my daily driver for a long time, and maintaining those two takes too much time for me. I will step down from maintenance those two and stop checking their issues and pull requests.

**Comment by stefonarch** [comment]
That's sad  news... thank you for your efforts all the time!

**Comment by tsujan** [comment]
Yes, it's sad news. Most LXQt components don't have a single maintainer, but your knowledge of qtermwidget was more than other devs'.

Don't hesitate to add comments or make PRs if you find time.

**Comment by yan12125** [comment]
Thank you both for kind words. It's a great experience to work with you all.

> Don't hesitate to add comments or make PRs if you find time.

Sure!

**Comment by AzumaHazuki** [comment]
Okay, soooo, does this mean we need another default terminal? Konsole is likely too tightly integrated into KDE, Alacritty pulls in Rust, Foot only works in Wayland...but the other "easy" terminal emulators are all GTK based and something like urxvt is really user-unfriendly out of the box.

**Reply by tsujan** [reply]
> does this mean we need another default terminal?

Of course not.
